pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -39,18 +39,18 @@ const (
 	maxAllowableNUMANodes = 8
 )
 
-//Manager interface provides methods for Kubelet to manage pod topology hints
+// Manager interface provides methods for Kubelet to manage pod topology hints
 type Manager interface {
-	//Manager implements pod admit handler interface
+	// Manager implements pod admit handler interface
 	lifecycle.PodAdmitHandler
-	//Adds a hint provider to manager to indicate the hint provider
-	//wants to be consoluted when making topology hints
+	// AddHintProvider adds a hint provider to manager to indicate the hint provider
+	// wants to be consoluted when making topology hints
```

Comment:
does consoluted here refers to https://www.thefreedictionary.com/consolute ? Maybe consulted?

```suggestion
	// wants to be consulted with when making topology hints
```